docs/en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Managing Security Issues
+summary: This document highlights how Silverstripe CMS team handles security issue.
```

Comment:
```suggestion
summary: This document highlights how the Silverstripe CMS team handles security issue.
```
Is "Silverstripe CMS team" the best way to refer to us? I don't think that term is ever used anywhere else.